static/app/gettingStartedDocs/apple/apple-ios.tsx

```diff
@@ -133,138 +243,241 @@ const getVerifyMetricsSnippetObjC = () => `
+                        {
+                          appDelegate: <code />,
+                        }
+                      )}
+                    </ListItem>
+                    <ListItem>
+                      {tct(
+                        'Add a new [phase: Upload Debug Symbols] phase to your [xcodebuild: xcodebuild] build script',
+                        {
+                          phase: <code />,
+                          xcodebuild: <code />,
+                        }
+                      )}
+                    </ListItem>
+                    <ListItem>
+                      {tct(
+                        'Create [sentryclirc: .sentryclirc] with an auth token to upload debug symbols (this file is automatically added to [gitignore: .gitignore])',
+                        {
+                          sentryclirc: <code />,
+                          gitignore: <code />,
+                        }
+                      )}
+                    </ListItem>
+                    <ListItem>
+                      {t(
+                        "When you're using Fastlane, it will add a Sentry lane for uploading debug symbols"
+                      )}
+                    </ListItem>
+                  </List>
+                ),
+                additionalInfo: tct(
+                  'Alternatively, you can also [manualSetupLink:set up the SDK manually].',
+                  {
+                    manualSetupLink: (
+                      <ExternalLink href="https://docs.sentry.io/platforms/apple/guides/ios/manual-setup/" />
+                    ),
+                    stepsBelow: <strong />,
+                  }
+                ),
+              },
+            ],
+          },
+        ]
+      : [
+          {
+            type: StepType.CONFIGURE,
+            description: (
+              <p>
                 {tct(
-                  'Add a new [phase: Upload Debug Symbols] phase to your [xcodebuild: xcodebuild] build script',
+                  'Make sure you initialize the SDK as soon as possible in your application lifecycle e.g. in your AppDelegate [appDelegate: application:didFinishLaunchingWithOptions] method:',
                   {
-                    phase: <code />,
-                    xcodebuild: <code />,
+                    appDelegate: <code />,
                   }
                 )}
-              </ListItem>
-              <ListItem>
+              </p>
+            ),
+            configurations: [
+              {
+                language: 'swift',
+                code: getConfigurationSnippet(params),
+              },
+              {
+                description: (
+                  <p>
+                    {tct(
+                      "When using SwiftUI and your app doesn't implement an app delegate, initialize the SDK within the [initializer: App conformer's initializer]:",
```

Comment:
I'm not sure exactly what's going on with `[initializer: App conformer's initializer]`, is that dynamically replaced with something else?